src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -420,6 +440,26 @@ static int _PaPulseAudio_ProcessAudio(PaPulseAudio_Stream *stream,
             PaUtil_SetOutputFrameCount( &stream->bufferProcessor,
                                         hostFramesPerBuffer );
 
+        }
+
+        hostFrameCount =
+            PaUtil_EndBufferProcessing( &stream->bufferProcessor,
+                                        &ret );
+
+        PaUtil_EndCpuLoadMeasurement( &stream->cpuLoadMeasurer,
+                                      hostFrameCount );
+
+        /*
+         * Idea behind this is that is there is output we have
```

Comment:
The beginning of this paragraph is unclear.

Reply:
Tried to change wording to be more exact